src/pca.jl

```diff
@@ -47,11 +47,27 @@ transform(M::PCA{T}, x::AbstractVecOrMat{T}) where {T<:Real} = transpose(M.proj)
 reconstruct(M::PCA{T}, y::AbstractVecOrMat{T}) where {T<:Real} = decentralize(M.proj * y, M.mean)
 
 ## show & dump
-
 function show(io::IO, M::PCA)
     print(io, "PCA(indim = $(indim(M)), outdim = $(outdim(M)), principalratio = $(principalratio(M)))")
 end
 
+function show(io::IO, ::MIME"text/plain", M::PCA)
+    print(io, "PCA(indim = $(indim(M)), outdim = $(outdim(M)), principalratio = $(principalratio(M)))")
+    ldgs = projection(M) * diagm(0 => sqrt.(M.prinvars))
+    rot = diag(ldgs' * ldgs)
+    ldgs = ldgs[:,sortperm(rot, rev=true)]
+    ldgs_signs = sign.(sum(ldgs, dims=1))
+    ldgs_signs[ldgs_signs .== 0] .= 1
+    ldgs = ldgs * diagm(0 => ldgs_signs[:])
+    print(io, "\n\nPattern matrix\n")
+    display(ldgs)
```

Comment:
It would be nice if this came out with columns and rows labeled but without access to the variable names that's not possible. In the current state the columns are the components in the same order as those in the following coeftable.

Reply:
Yes, that's annoying. For now maybe print a `CoefTable` object without row names. If you pass an empty vector as row names, numbers will automatically be generated. That way at least the names of PCs will be consistent with the table below.

Also, how about printing the second table first? That's the most important result I think.